ecosystem/resources/members/qiskit-ibm-provider.toml

```
@@ -1,6 +1,6 @@
 name = "Qiskit IBM Provider"
 url = "https://github.com/Qiskit/qiskit-ibm-provider"
-description = "This project contains a provider that allows accessing the IBM Quantum systems and simulators."
+description = "Access IBM Quantum systems and simulators."
```

Since cloud simulators will be sunset in early 2024, should we scrub "and simulators" from this description now, or would you prefer to wait till they are gone?

---

Good call, we'll definitely forget to update this when that happens 😆

```suggestion
description = "Access IBM Quantum systems."
```